ios/RNSScreenStackHeaderConfig.mm

```diff
@@ -495,7 +495,8 @@ + (void)updateViewController:(UIViewController *)vc
       // iOS 12 cancels swipe gesture when direction is changed. See #1091
       navctr.view.semanticContentAttribute != config.direction) {
     navctr.view.semanticContentAttribute = config.direction;
-    navctr.navigationBar.semanticContentAttribute = config.direction;
+    [[UIButton appearance] setSemanticContentAttribute:config.direction];
+    [[UIView appearance] setSemanticContentAttribute:config.direction];
```

Comment:
It sets `RTL` on all buttons and views in the app this way?

Reply:
I am not sure about that. Probably not, as setting the UIView appearance here does not change the direction of search bar contents - it still required to set the direction for those subviews individually.

Reply:
I'm reading now into RTL support in UIKit & it seems it does. I'll post a review once I'm familiar a bit more with this mechanism in UIKit.

Reply:
Ended up narrowing targets to UIViews and UIButtons contained in `navctr` https://github.com/software-mansion/react-native-screens/pull/2185/commits/a2dcafc57f947d023341d1614f7e7fca3d4aaecc

Reply:
Since practically the whole App should be under the navigation structure, it does pretty much the same 😅